tests/test_qube_coverage.py

```
@@ -0,0 +1,1622 @@
+        # Test as_int edge cases
+        ##################################################################################
+        a = Scalar([1.5, 2.5, 3.5])
+        b = a.as_int()
+        self.assertTrue(b.is_int())
+
+        ##################################################################################
+        # Test as_bool edge cases
+        ##################################################################################
+        # Test with builtins=True and scalar (line 2423-2424)
+        a = Scalar(1.)
+        Qube.prefer_builtins(True)
```

Ruff (0.14.7)

Boolean positional value in function call

(FBT003)